src/govuk-prototype-kit/govuk-prototype-kit.config.unit.test.mjs

```diff
@@ -0,0 +1,163 @@
+import configFn from './govuk-prototype-kit.config.mjs'
```

Comment:
Assuming we're going to use this to write out a JSON file at the point we build the package, would it make sense to make assertions about the generated JSON (and that it matches paths that exist in the package) rather than testing the logic in this file?

(Or both?)

Reply:
Was going to add a snapshot for the JSON, but similarly to https://github.com/alphagov/govuk-frontend/pull/2851#discussion_r1005825252 that'll break the test every time a new component is added

What would you prefer?

Reply:
@36degrees Ah so the old implementation is a static JSON file that doesn't do anything from the list above

Would you consider these non-blocking enhancements for another PR?

Reply:
That's a fair point, although the macros are a new addition to the config file.

Happy to treat as an enhancement.

Reply:
Brill, thanks

You still happy with the static `nunjucksMacros` array in the tests?

It was the quickest (dumb) way for me to say "This still does what it used to"

Reply:
Yep, I think so